docs/readmes/faq/faq_magma.md

```diff
@@ -79,9 +79,9 @@ This section lists some of the commonly asked questions related to Magma operati
     - From the popup window, select tab **COMMANDS**. Then, check section **Reboot**.
     - Click on **Reboot** to reboot the AGW node.
 
-### Does AGW supports IPv6?
+### Does AGW support IPv6?
 
-- IPv6 and dual stack IPv4V6 session support are currently under active development and we are working towards making this feature ready as part of release 1.4.
+- IPv6 and dual stack IPv4v6 session support is implemented for Magma v1.7 and later.
```

Comment:
Should we also change that in `docs/docusaurus/versioned_docs/version-1.7.0/faq/faq_magma.md` ?

Reply:
I think that's a good idea

Reply:
Done. I also updated this part for the versioned docs from 1.4 onward, to reflect the active development status (it doesn't make sense to say it will be released in 1.4 in the 1.5 docs, for example)